src/ZEO/tests/testZEO.py

```diff
@@ -1636,7 +1724,9 @@ def pack(self, t=None, referencesf=None):
         ZEO.ClientStorage._check_blob_cache_size(self.blob_dir, 0)
 
 def test_suite():
-    suite = unittest.TestSuite()
+    suite = unittest.TestSuite((
```

Comment:
Is `suite = unittest.defaltTestLoader.loadTestsFromName(__name__)` an option here to avoid repetition?

Reply:
Heh, have look at that `test_suite` function -- I don't have enough in my gumption reservoir to do anything other than the most minimal change to get my tests to run.

Reply:
Hehe, fair enough. I didn’t want to deal with it either, I was just trying to pass the buck 😄